db/migrate/20201020154112_change_default_value_of_second_factor_attempt_count.rb

```diff
@@ -0,0 +1,15 @@
+class ChangeDefaultValueOfSecondFactorAttemptCount < ActiveRecord::Migration[6.0]
+  def change
+    reversible do |dir|
+      dir.up do
+        change_column_default(:users, :second_factor_attempts_count, from: nil, to: 0)
+        User.update_all(second_factor_attempts_count: 0)
+      end
+
+      dir.down do
+        User.update_all(second_factor_attempts_count: nil)
```

Comment:
I'd remove this line. Restoring a database default I can get behind but voluntarily restoring buggy data, not so much.